.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2.1
+
+orbs:
+  plugin-ci: mattermost/plugin-ci@0.1.0
+
+workflows:
+  version: 2
+  ci:
+    jobs:
+      - plugin-ci/lint
+      - plugin-ci/test
+      - plugin-ci/build
```

Comment:
This has to be the config file from https://github.com/mattermost/mattermost-plugin-demo/blob/master/.circleci/config.yml, otherwise github releases won't get automatically created.